recipes/python-chromedriver-binary/download-chromedriver.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+import subprocess
+
+pkg_version = os.environ['PKG_VERSION']
+driver_version = "{0}.{1}".format(*pkg_version.split("."))
+if sys.platform == 'win32':
+    chromedriver_pkg = 'chromedriver_win32.zip'
+elif sys.platform == 'darwin':
+    chromedriver_pkg = 'chromedriver_mac64.zip'
+else:
+    chromedriver_pkg = 'chromedriver_linux64.zip'
+
+
+download_url = 'https://chromedriver.storage.googleapis.com/{0}/{1}'.format(driver_version, chromedriver_pkg)
+
+subprocess.call(['curl', download_url, '-o', 'chromedriver.zip'], shell=True)
```

Comment:
There are subtle and annoying differences in the command line handling of `subprocess.call` when `shell=True` between Windows and Linux IIRC, try passing just a string:

```python
subprocess.call('curl "{}" -o chromedriver.zip'.format(download_url), shell=True)
```